pkg/storaged/content-views.jsx

```diff
@@ -139,19 +148,29 @@ function create_tabs(client, target, options) {
     let warnings = client.path_warnings[target.path] || [];
     if (content_block)
         warnings = warnings.concat(client.path_warnings[content_block.path] || []);
+    if (lvol)
+        warnings = warnings.concat(client.path_warnings[lvol.path] || []);
 
     const tab_actions = [];
     const tab_menu_actions = [];
     const tab_menu_danger_actions = [];
 
     function add_action(title, func) {
-        tab_actions.push(<StorageButton onlyWide key={title} onClick={func}>{title}</StorageButton>);
-        tab_menu_actions.push({ title, func, only_narrow: true });
+        if (tab_actions.length == 0) {
+            tab_actions.push(<StorageButton onlyWide key={title} onClick={func}>{title}</StorageButton>);
+            tab_menu_actions.push({ title, func, only_narrow: true });
+        } else {
+            add_menu_action(title, func);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17226-20230918-142947-d0c778b0-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)